README.md

```diff
@@ -11,9 +11,10 @@
 
 The [Cloudsmith](https://cloudsmith.io) Command Line Interface (CLI) is a Py2/Py3 text-based interface to the [API](https://api.cloudsmith.io). This allows users, machines and other services to access and integrate smoothly with Cloudsmith without requiring explicit plugins or tools. [Be awesome. Automate Everything](https://cloudsmith.com/company/the-tao-of-cloudsmith/).
 
-The following (very out of date) GIF demonstrates a small slice of the CLI - [View the full video on YouTube](https://youtu.be/8nlstYU9J5M):
+The following asciienema video demonstrates some of the CLI commands:
+[![asciicast](https://asciinema.org/a/DkNXQWQGBjWkfGPAkDAPNz7xe.svg)](https://asciinema.org/a/DkNXQWQGBjWkfGPAkDAPNz7xe)
```

Comment:
looks like the video is uploaded to a personal account; should this be on a cloudsmith account instead?

Reply:
This is a good point - the account is under dmckinney@cloudsmith.io

Reply:
Account updated to "Cloudsmith" - still pulls my avatar from elsewhere though

Reply:
will others have access to the shared account to upload also?

Reply:
They will need to, so I'll need to change the email address on the account to engineering@ or similar - no passwords on asciinema, all logins done by one time links sent via email. Suggestions for the email account to use?

Reply:
`engineering@cloudsmith.io`probably makes the most sense to avoid additional setup. we can always change it later if it gets noisy

Reply:
Shouldn't get too bad, I'll change that now.

Reply:
Account name updated to "Cloudsmith" account email set to engineering@cloudsmith.io - need a one time link sent to that address to log in 👍 .